### ISSUE TITLE
Hook up a twitter button that does something similar to what Fb does on the share pages

### ISSUE THREAD

**Issue by threeqube** [mozilla/openbadges-backpack#770]
with editable pre-canned message linking to single badge page url as referenced here:
https://github.com/mozilla/openbadges/issues/689#issuecomment-14789754

**Comment by mattdigitalme** [comment]
@threeqube @auralon Does sharing happen from the Backpack as much as expected, or is sharing done more from the badge issuing platform? If future of BP becomes more of a central store rather than public facing view, do we need further sharing integration work?

**Comment by auralon** [comment]
I think the ability to share is always nice.  Users who collect badges do so to ultimately show them off, and that isn't always in a collection context.

**Comment by kayaelle** [comment]
Hey @mattdigitalme - I'd caution against referring to the backpack as a central store unless the intention of the backpack is to actually be the host of the json data for the badges. If the functionality continues as is which is linking to the hosted data and urls, it's critical that all parties understand that the hosting (of data and of private keys for signed assertions) continues to be the responsibility of the issuer. There is often confusion about this in the community and an essential point to communicate. Thanks!

**Comment by mattdigitalme** [comment]
Fair point @kayelle - central store to me likely means something different in various contexts - apologies for confusion

**Comment by kayaelle** [comment]
No apologies needed! Just want to ensure that it's kept in mind when communicating with the community.
